setup.py

```diff
@@ -0,0 +1,67 @@
+"""
+Copyright 2018 SuperDARN
+
+setup.py
+2018-11-05
+To setup pyDARN as a third party library. Include installing need libraries for
+running the files.
+
+author:
+Marina Schmidt
+"""
+
+from distutils.core import setup
```

Comment:
I don't think this is needed? It's immediately overwritten by the next line, right?